scripts/src/main/resources/scripts/command/oc

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+OPENSHIFT_HOME="${DEVON_IDE_HOME}/software/oc/"
+
+# openshift installation
+function doSetup(){
+
+  if [ ! -d "${OPENSHIFT_HOME}" ] || [ "${1}" != "silent" ]
+  then
+    # Getting leatest release
+    doEcho "Getting latest release..."
+    local MAJOR_VERSION
+    local LATEST_VERSION
+    MAJOR_VERSION=$(curl "https://mirror.openshift.com/pub/" | awk -F "openshift-v" '/openshift-v/ {print $2}' | sort -r | grep -o -E "[0-9]+" | uniq | head -1)
+    LATEST_VERSION=$(curl "https://mirror.openshift.com/pub/openshift-v${MAJOR_VERSION}/clients/ocp/latest/" | awk -F "openshift-client-windows-" '/openshift-client-windows-/ {print $2}' | head -1 | awk -F'.zip' '{print $1}')
```

Comment:
This link only work for me for version4.
https://mirror.openshift.com/pub/openshift-v3/clients/ocp/latest/
For v1,v2,v3 I get File not found error.
I think we should set this to Version v4 and remove MAJOR_VERSION.